app/models/annotation/Annotation.scala

```diff
@@ -366,51 +365,48 @@ class AnnotationDAO @Inject()(sqlClient: SQLClient, annotationLayerDAO: Annotati
-                                  (select #$columns
-                                   from #$existingCollectionName
-                                   where _user = $userId and typ = '#$typ' and state = '#${AnnotationState.Active}' and #$accessQuery) a
-                                  join webknossos.teams t on a._team = t._id where #$excludeTeamsQ) q
+                                  (select $columns
+                                   from $existingCollectionName
+                                   where _user = $userId and typ = $typ and state = ${AnnotationState.Active} and $accessQuery) a
+                                  join webknossos.teams t on a._team = t._id where $excludeTeamsQ) q
                          """.as[Int])
       count <- countList.headOption
     } yield count
 
   def countActiveByTask(taskId: ObjectId, typ: AnnotationType)(implicit ctx: DBAccessContext): Fox[Int] =
     for {
       accessQuery <- readAccessQuery
-      countList <- run(
-        sql"""select count(*) from #$existingCollectionName
-              where _task = $taskId and typ = '#$typ' and state = '#${AnnotationState.Active}' and #$accessQuery"""
-          .as[Int])
+      countList <- run(q"""select count(*) from $existingCollectionName
+              where _task = $taskId and typ = $typ and state = ${AnnotationState.Active} and $accessQuery""".as[Int])
       count <- countList.headOption
     } yield count
 
   def countAllForOrganization(organizationId: ObjectId): Fox[Int] =
     for {
       countList <- run(
-        sql"select count(a._id) from #$existingCollectionName a join webknossos.users_ u on a._user = u._id where u._organization = $organizationId"
+        q"select count(a._id) from $existingCollectionName a join webknossos.users_ u on a._user = u._id where u._organization = $organizationId"
           .as[Int])
       count <- countList.headOption
     } yield count
 
   // update operations
 
   def insertOne(a: Annotation): Fox[Unit] = {
-    val viewConfigurationStr: Option[String] = a.viewConfiguration.map(Json.toJson(_).toString)
-    val insertAnnotationQuery = sqlu"""
+    val insertAnnotationQuery = q"""
         insert into webknossos.annotations(_id, _dataSet, _task, _team, _user, description, visibility,
                                            name, viewConfiguration, state, statistics, tags, tracingTime, typ, othersMayEdit, created, modified, isDeleted)
         values(${a._id}, ${a._dataSet}, ${a._task}, ${a._team},
-         ${a._user}, ${a.description}, '#${a.visibility}', ${a.name},
-         #${optionLiteral(viewConfigurationStr.map(sanitize))},
-         '#${a.state}', '#${sanitize(a.statistics.toString)}',
-         '#${writeArrayTuple(a.tags.toList)}', ${a.tracingTime}, '#${a.typ}',
+         ${a._user}, ${a.description}, ${a.visibility}, ${a.name},
+         ${a.viewConfiguration},
+         ${a.state}, ${a.statistics},
+         ${a.tags}, ${a.tracingTime}, ${a.typ},
```

Comment:
This would be neat:
```
SqlToken.insert("webknossos.annotations")(
  "_id" -> a._id,
  "_dataSet" -> a._dataSet,
  ...
).sql == "INSERT INTO webknossos.annotations(_id, _dataSet, ...) VALUES (?,?,...)"
```